key-vault/hashicorp-key-vault/pom.xml

```diff
@@ -16,12 +16,6 @@
                 <artifactId>spring-vault-core</artifactId>
                 <version>2.1.1.RELEASE</version>
             </dependency>
-
-            <dependency>
```

Comment:
Does this dependency need to be kept in the hashicorp dependency management for creation of the sharded jar?

Reply:
it should be included in the fat jar as long as it's specified in dependencies